static/docker-compose.yml

```diff
@@ -60,6 +60,8 @@ services:
       PG_CORE_DB: ${PG_CORE_DB:?err}
       PG_RUNBOOKS_ADMIN_PASSWORD: ${PG_RUNBOOKS_ADMIN_PASSWORD:?err}
       PG_RUNBOOKS_ADMIN_USER: ${PG_RUNBOOKS_ADMIN_USER:?err}
+      PG_RUNBOOKS_RW_PASSWORD: ${PG_RUNBOOKS_RW_PASSWORD:?err}
+      PG_RUNBOOKS_RW_USER: ${PG_RUNBOOKS_ADMIN_USER:?err}
```

Comment:
is the password correct?

Reply:
I think so. This is wiring up the environment variable so that it will be received by the docker container. The actual password will be set in an environment variable on each host environment.

Reply:
```suggestion
      PG_RUNBOOKS_RW_USER: ${PG_RUNBOOKS_RW_USER:?err}
```

Reply:
Ahh I see. Yes good catch!